src/symfony/harness/attributes/docker.yml

```diff
@@ -0,0 +1,9 @@
+attributes:
+  services:
+    php-base:
+      environment:
+        APP_DEBUG: "0"
+        APP_ENV: "= (@('app.mode') == 'development' ? 'dev' : 'prod')"
+      environment_secrets:
+        APP_SECRET: = @('symfony.secret')
+        DATABASE_URL: = 'mysql://' ~ @('database.user')  ~ ':' ~ @('database.pass')  ~ '@' ~ @('database.host') ~ ':' ~ @('database.port')  ~ '/' ~ @('database.name')  ~ '?serverVersion=' ~ @('database.platform_version')
```

Comment:
could be worth a skeleton config/...yml with:
```
attributes:
  env(DATABASE_URL): "%env(DATABASE_PLATFORM)%://%env(DATABASE_USER)%:%env(DATABASE_PASSWORD)%@%env(DATABASE_HOST)%:%env(DATABASE_PORT)%/%env(DATABASE_NAME)%?serverVersion=%env(DATABASE_VERSION)%

Reply:
Great idea. Done!

One caveat is that .env can't have DATABASE_URL in, as it takes precedence over the config/*/*.yaml